diff --git a/lib/config/opApplicationConfiguration.class.php b/lib/config/opApplicationConfiguration.class.php
index 5a288dd..a64d7ca 100644
--- a/lib/config/opApplicationConfiguration.class.php
+++ b/lib/config/opApplicationConfiguration.class.php
@@ -527,24 +527,6 @@ abstract class opApplicationConfiguration extends sfApplicationConfiguration
     require_once 'Auth/OpenID/SReg.php';
   }
 
-  public function setCacheDir($cacheDir)
-  {
-    $newCacheDir = $cacheDir.DIRECTORY_SEPARATOR;
-    if (is_callable('posix_getuid'))
-    {
-      $userinfo = posix_getpwuid(posix_getuid());
-      $newCacheDir .= $userinfo['name'];
-    }
-    else
-    {
-      $newCacheDir .= php_sapi_name();
-    }
-
-    sfConfig::set('sf_cache_dir', $newCacheDir);
-
-    parent::setCacheDir($newCacheDir);
-  }
-
   public function generateAppUrl($application, $parameters = array(), $absolute = false)
   {
     list($route, $parameters) = sfContext::getInstance()->getController()
diff --git a/lib/config/opProjectConfiguration.class.php b/lib/config/opProjectConfiguration.class.php
index de1b040..bbb7490 100644
--- a/lib/config/opProjectConfiguration.class.php
+++ b/lib/config/opProjectConfiguration.class.php
@@ -115,6 +115,24 @@ class opProjectConfiguration extends sfProjectConfiguration
     $this->setupProjectOpenPNEDoctrine($manager);
   }
 
+  public function setCacheDir($cacheDir)
+  {
+    $newCacheDir = $cacheDir.DIRECTORY_SEPARATOR;
+    if (is_callable('posix_getuid'))
+    {
+      $userinfo = posix_getpwuid(posix_getuid());
+      $newCacheDir .= $userinfo['name'];
+    }
+    else
+    {
+      $newCacheDir .= php_sapi_name();
+    }
+
+    sfConfig::set('sf_cache_dir', $newCacheDir);
+
+    parent::setCacheDir($newCacheDir);
+  }
+
   protected function setOpenPNEConfiguration()
   {
     $opConfigCachePath = sfConfig::get('sf_cache_dir').DIRECTORY_SEPARATOR.'config'.DIRECTORY_SEPARATOR.'OpenPNE.yml.php';
